test/karma.conf.js

```diff
@@ -69,6 +69,7 @@ module.exports = function(config) {
       'lib/modernizr.custom.80690.js',
       // application files
       'app/**/*.js',
+      'app/common/**/*.js',
```

Comment:
@psbrandt It should but I faced many issues while testing, it always reported that a "module" on that folder was not being loaded. 
So in the karma github issues page and stackoverflow forum they stated that was a problem for the karma version and the solution will be rolled out in the next release.